### ISSUE TITLE
RxContainer

### ISSUE THREAD

**Issue by DevAndArtist** [RxSwiftCommunity/contributors#25]
- [x] replace copyright with "Copyright (c) RxSwiftCommunity" in code
- [x] replace copyright in LICENSE
- [ ] replace copyright in README
- [ ] transfer repository ownership (once you're a contributor)
- [ ] add project to https://github.com/RxSwiftCommunity/rxswiftcommunity.github.io
- [ ] fork the repo back to your personal account
- [ ] update the remote URL of the Podspec (if any)
- [ ] update CI badge
- [ ] update CI settings (what exactly?)

https://github.com/DevAndArtist/RxContainer

**Comment by ashfurrow** [comment]
Cool! I've invited you to the organization, let me know if I can be of further help!

**Comment by DevAndArtist** [comment]
@ashfurrow Hi there, thank you. I want to add Travis-CI to this project. How should I update CI settings when the project is transferred?

**Comment by ashfurrow** [comment]
Hmm, not entirely sure. I usually have to update the badge URL and the rest "just works" – we're here to help if things go wrong!

**Comment by freak4pc** [comment]
Just out of context of this specific ticket but I think it would be highly beneficial if your repo explains exactly *what* it does and what it aims to solve... It immediately show code with no explanation and that might be confusing to people browsing through the RxCommunity repo

**Comment by DevAndArtist** [comment]
@freak4pc Yeah, well there is still some work to do and some more documentations. It's just there so it does not look empty at this point.

**Comment by ashfurrow** [comment]
Hi @DevAndArtist! Anything I can to to help follow-up on this? Just let me know 😃
